src/locales/en.json

```diff
@@ -377,11 +365,14 @@
       "export": "Export data",
       "price_list": "View all price lists"
     },
+    "change_column_title": "Month over month change",
```

Comment:
Probably want to rename this to something more generic, like month_over_month_change?

Reply:
Done